app/screens/MonthlyBudget/MonthlyBudgetScreen.tsx

```diff
@@ -5,13 +5,17 @@ import { Ionicons } from "@expo/vector-icons";
 import * as WebBrowser from "expo-web-browser";
 import { useDispatch, useSelector } from "react-redux";
 
+import { budget } from "ducks";
+
+import { t } from "utils";
+
+import { Colors } from "style";
+
+import { navigate } from "navigation";
+
 import { Text, Button } from "../../components";
```

Comment:
Should be `import { Text, Button } from "components";`

Reply:
I'll take a look at this tonight or tomorrow :)